css/properties/break-after.json

```diff
@@ -82,93 +28,365 @@
               "ie": {
                 "version_added": "10"
               },
-              "opera": {
-                "version_added": "11.1",
-                "version_removed": "12.1"
-              },
+              "opera": [
+                {
+                  "version_added": "37"
+                },
+                {
+                  "version_added": "11.1",
+                  "version_removed": "12.1"
+                }
+              ],
               "opera_android": {
                 "version_added": null
               },
               "safari": {
-                "version_added": false
+                "version_added": "10"
               },
               "safari_ios": {
-                "version_added": false
+                "version_added": "10"
               },
               "samsunginternet_android": {
-                "version_added": false
+                "version_added": "5.0"
               },
               "webview_android": {
-                "version_added": false
+                "version_added": "50"
               }
             },
             "status": {
               "experimental": false,
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "column": {
+            "__compat": {
+              "description": "<code>column</code> and <code>avoid-column</code>",
+              "support": {
+                "chrome": {
+                  "version_added": "50",
+                  "notes": "Supports <code>column</code> but not <code>avoid-column</code>."
```

Comment:
Thanks for this @rachelandrew! Just one area I had questions about.

* If this features includes `column` and `avoid-column`, then I think this entry (and the IE entry below) needs to have `"partial_support": true` or we need to consider splitting up `column` and `avoid-column` into separate features
* If Chrome only supports `column` then I'm skeptical that Chrome for Android and Webview have support for `avoid-column`. Is that something we can substantiate? Or maybe it doesn't matter if the feature is split in two

Reply:
@ddbeck you are right, so I guess we could split them or just put a note as I have for the other one. I am fine with that either way, do you have a preference?

Reply:
I guess I prefer splitting them in two. Sorry for the delay on answering your question, @rachelandrew.

Reply:
@ddbeck I've just pushed a commit splitting these.

Reply:
Thank you!